src/webpack/webpack.common.config.ts

```diff
@@ -1,6 +1,7 @@
 import path from 'path';
 
 import CaseSensitivePathsPlugin from 'case-sensitive-paths-webpack-plugin';
+// eslint-disable-next-line import/default
 import CopyWebpackPlugin from 'copy-webpack-plugin';
```

Comment:
Den klager på `No default export found in imported module "copy-webpack-plugin"`. Fra dokumentasjonen deres ser det egentlig ut som man skal bruke require og ikke import, men usikker på om vi vil det. Hva tenker dere andre?
https://www.npmjs.com/package/copy-webpack-plugin/v/6.3.1

Reply:
Fungerer det bedre hvis du kjører en slik variant? Det hadde jo vært fint å kunne holde oss kun til imports hvis vi kan det, uten at jeg har noen sterke meninger her 🤔 
`import * as CopyWebpackPlugin from 'copy-webpack-plugin'`

Reply:
Nei da får jeg problemer med hvordan den blir brukt 😢